notebooks/MAST/ultrafaint_dwarf_galaxy_search/ultrafaint_dwarf_galaxy_search.ipynb

```diff
@@ -0,0 +1,1247 @@
+{
```

Comment:
<p>Please transform pixels coordinates to RA/Dec coordinates and show how to do that. Please do so all other sky images below.</p><p>I also wonder if you could find another image of Leo from literature and compare the PS1 image with it. If you cannot find, no worries.</p><p> </p><p>I also suggest to show some basic fits information like HDU information and show some headers like coordinates and object names . </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
I can't find any images of Leo I, I can only find Leo IV and Leo P.